db/migrate/20180926021039_add_devise_to_admin_users.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class AddDeviseToAdminUsers < ActiveRecord::Migration
```

Comment:
oh i see, it is from devise, i would suggest you to use something more generic like `user` or `account` ad the end be an admin is just a modifier like a rol